packages/compile-common/src/index.ts

```diff
@@ -1,2 +1,62 @@
 import { Profiler } from "./profiler";
 export { Profiler };
+
+type Compilation = {
+  sourceIndexes: string[];
+  contracts: CompiledContract[];
+  compiler: {
+    name: string;
+    version: string;
+  };
+};
+
+type CompilerResult = Compilation[];
+
+type CompiledContract = {
+  contractName: string;
+  contract_name: string;
```

Comment:
let's get rid of this from the type and make sure we just translate before creating something of this type.

Reply:
So just to make sure I understand, we're going to get rid of `contract_name` which WILL exist on the saved artifact type.

Reply:
Yep, but that's fine, because this type doesn't represent an artifact